data/campaigns/Eastern_Invasion/scenarios/09_Castle_in_the_Ice.cfg

```diff
@@ -0,0 +1,1425 @@
+#textdomain wesnoth-ei
+[scenario]
+    id=09_Castle_in_the_Ice
+    name= _ "Castle in the Ice"
+    map_file=09_Castle_in_the_Ice.map
+    victory_when_enemies_defeated=no
+    next_scenario=10_Dark_Sanctuary
+    turns=-1
+
+    #
+    # -------WILDLANDS GOAL NOTES:
+    # this is a long campaign, and I wanted to avoid it feeling tedious
+    # so, the wildlands arc (placed at the campaign's middle) has a lot of unusual mechanics, to try and make these scenarios feel different from normal scenarios
+    # S09 has "frostbite" and several events, S10 has a split army with many events, and S11 is heavily scripted.
+    # the income changes hopefully put emphasis on gold rather than army, and being frugal rather than rushing for an early finish (there are no early finish bonuses)
+    #
+
+    #
+    # -------WILDLANDS INCOME NOTES:
+    # wildlands scenarios all have 100% carryover and 0 starting gold (except for S08 with some starting gold, and S12 with 40% carryover)
+    # in S09, Dacyn warns you to conserve your gold, but even if you spend recklessly it's extremely difficult to make a subsequent scenario unwinnable:
+    #     S08 (Xenophobia) requires you to end with at least 300 gold
+    #     S09 (Castle In The Ice) needs few recruits and only has a single-tile castle. You'd have to try pretty hard to go broke.
+    #             plus there are several gold pickups lying around
+    #             this scenario has no time limit (i.e. no early finish bonus), but few villages and continual cold damage, to hopefully prevent abuse
+    #     S10 (Dark Sanctuary) is a bit of a gold-eater, but even if you spend everything now, S12 is still winnable with 0 gold
+    #             there's also gold just inside the gate (wyvern), ensuring that even if you spend everything immediately, you still have enough gold to resist the orcs
+    #             this scenario has no time limit (i.e. no early finish bonus), but few villages and increasing enemy spawns, to hopefully prevent abuse
+    #     S11 (Captured) has no need for recruiting
+    #             this scenario has no time limit so you can farm infinite xp, but without any gold you won't be able to bring leveled recalls through S12
+    #             so it's in the player's best interest to rush the exit as quickly as possible
+    #     S12 (Evacuation) is beatable without any recruiting - just run straight for the bridge
+    #             but S08-S11 gold conservation pays off here, letting you recruit a large army and potentially save all your leveled recalls
+    #
+
+    {WINTER_SCHEDULE}
+
+    {SCENARIO_MUSIC       suspense.ogg}
+    {EXTRA_SCENARIO_MUSIC nunc_dimittis.ogg}
+
+    [story]
+        [part]
+            story=_ "As Gweddry and his band continued north, winter tightened its grip on the wildlands. The days began to lengthen and the air swiftly grew chill, making the already challenging mountains even more perilous to traverse."
```

Comment:
Uh, if winter is setting in, shouldn't it read 'nights began to lengthen' or 'days began to shorten'?

Reply:
Oops yes you're right.  Thanks for noticing.